treeherder/etl/buildbot.py

```diff
@@ -59,6 +59,14 @@
         }
     },
     {
+        'regex': re.compile(r'^graphene.*_linux64', re.IGNORECASE),
```

Comment:
I believe this needs to be higher up, since the regex above will win (they are executed in order listed here).